fe/fe-core/src/main/java/com/starrocks/common/ErrorCode.java

```diff
@@ -333,6 +333,8 @@ public enum ErrorCode {
             "%s column: %s has no matching %s column"),
     ERR_FAILED_WHEN_INSERT(5609, new byte[] {'2', '2', '0', '0', '0'}, "Failed when executing INSERT : '%s'"),
     ERR_LOAD_HAS_FILTERED_DATA(5610, new byte[] {'2', '2', '0', '0', '0'}, "Insert has filtered data : %s"),
+    ERR_PARSE_ERROR(5611, new byte[] {'2', '2', '0', '0', '0'},
```

Comment:
The name is too general.
You can name it as `ERR_INPUT_DATA_PARSE_ERROR`

Reply:
fixed. ERR_LOAD_DATA_PARSE_ERROR